lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -42,6 +42,8 @@ def find_and_load(path: nil)
         #
         # @quality :reek:ControlParameter
         def find(path: nil, current: Pathname.pwd, home: Pathname.new(Dir.home))
+          # TODO: Here we need to return the configuration path as well, that means changing find_by_dir and find_in_dir
+          # as well.
```

Comment:
Isn't the configuration path exactly what it is returning?

Reply:
Yeah, I'm not fully sure anymore why I wrote down this TODO, thanks for the pointer 😆